rho/translation.py

```diff
@@ -19,6 +19,7 @@
 def get_translation():
     """Obtains the locale based translation setup for rho"""
     if hasattr(T, 'ugettext'):
+        # pylint: disable=no-member
```

Comment:
This disable could be avoided by writing something like this:

```python
_ getattr(T, 'ugettext', T.gettext)
```

If `gettext` is not available then another getattr can be used.